frontend/packages/console-app/src/components/user-preferences/perspective/usePreferredPerspective.ts

```diff
@@ -0,0 +1,10 @@
+import { useUserSettings } from '@console/shared';
+
+const PREFERRED_PERSPECTIVE_USER_SETTING_KEY: string = 'console.preferredPerspective';
```

Comment:
You don't need add unambiguous types when defining a string const.

This is maybe a good place for a const for "latest". Maybe we can change then the value to `%LATEST%` similar to ALL_NAMESPACES_KEY to ensure that we don't have a problem with a project which is called "latest". :)

```suggestion
const PREFERRED_PERSPECTIVE_USER_SETTING_KEY = 'console.preferredPerspective';
const PREFERRED_PERSPECTIVE_USER_SETTING_LATEST = '%LATEST%';
```